src/test/java/org/optsolvx/tests/CommonsMathSolverTest.java

```diff
@@ -194,25 +194,4 @@ void testNumericalPrecision() {
         assertEquals(1.0 / 3.0, sol.variableValues[0], 1e-8, "x should be close to 1/3");
```

Comment:
Please write `1d` instead of `1.0` because this is clear. You may alternatively write `1f` if the data type `float` is sufficient for your literal numbers.